tests/extract/conftest.py

```diff
@@ -181,11 +174,13 @@ def env_setup():
     ]
     for a in attempt:
         try:
-            MySQL(host=a)
-            os.environ["DATABASE_URL"] = a
+            with MySQL(host=a):
+                os.environ["DATABASE_URL"] = a
```

Comment:
I suspected that the database being open maybe causing my deadlock.  Closed database, but no change.  Still a good change though.